LEGAL

```diff
@@ -446,7 +446,7 @@ mentioned below.
 
   >>>
     A C-program for MT19937, with initialization improved 2002/2/10.::
-    Coded by Takuji Nishimura and Makoto Matsumoto.  
+    Coded by Takuji Nishimura and Makoto Matsumoto.
```

Comment:
This space is intentional to break the line.

Reply:
Thanks, It's accidental commit.